tests/test-cases/cli/inspect/multi-attribute-resource-values/multi-attribute-duplicate-labels.csv-metadata.json

```diff
@@ -338,6 +359,63 @@
                     "@value": "Multi Attribute"
                 }
             ],
+            "http://www.w3.org/ns/prov#wasGeneratedBy": [
+                {
+                    "@id": "multi-attribute.csv#csvcubed-build-activity"
+                }
+            ],
+            "http://www.w3.org/ns/dcat#distribution": [
+                {
+                    "@id": "multi-attribute.csv#csvqb"
+                }
+            ]
+        },
+        {
+            "@id": "multi-attribute.csv#csvqb",
+            "@type": [
+                "http://purl.org/linked-data/cube#Attachable",
+                "http://www.w3.org/ns/dcat#Distribution",
+                "http://www.w3.org/ns/dcat#Resource",
+                "http://purl.org/linked-data/cube#DataSet",
+                "http://www.w3.org/2000/01/rdf-schema#Resource"
+            ],
+            "http://purl.org/dc/terms/creator": [
+                {
+                    "@id": "https://www.gov.uk/government/organisations/office-for-national-statistics"
+                }
+            ],
+            "http://purl.org/dc/terms/identifier": [
+                {
+                    "@value": "Multi Attribute"
+                }
+            ],
+            "http://purl.org/dc/terms/issued": [
+                {
+                    "@type": "http://www.w3.org/2001/XMLSchema#dateTime",
+                    "@value": "2023-04-04T11:09:03.509468"
+                }
+            ],
+            "http://purl.org/linked-data/cube#structure": [
+                {
+                    "@id": "multi-attribute.csv#structure"
+                }
+            ],
+            "http://www.w3.org/2000/01/rdf-schema#label": [
+                {
+                    "@language": "en",
+                    "@value": "Multi Attribute"
+                }
+            ],
+            "http://www.w3.org/ns/dcat#isDistributionOf": [
+                {
+                    "@id": "multi-attribute.csv#dataset"
+                }
+            ],
+            "http://www.w3.org/ns/prov#wasDerivedFrom": [
```

Comment:
The data was derived form the source csv, not csvcubed. The actual realtionship would be

```ttl
#csvqb provo:wasDerivedFrom <source csv uri>;
    provo:wasGeneratedBy <multi-attribute.csv#csvcubed-build-activity>
```

The csvcubed-build-activity declaration elsewhere is on point.